src/test/javascript/spec/screenshot.ts

```diff
@@ -0,0 +1,106 @@
+import { browser, $, expect } from '@wdio/globals';
+import { WdioCheckElementMethodOptions } from '@wdio/visual-service/dist/types';
+import * as path from 'path';
+
+const BASE_URL = 'http://localhost:9000';
+const DATABASE_EMULATOR_URL = 'http://127.0.0.1:9095/?ns=mock-data';
```

Comment:
does `ns=mock-data` have to match the name of the json file that we have in `test/javascript/firebase/mockb-data.json`?

Reply:
> I feel like this should be in an .env file

I think we as the scope grows and we find ourselves needing to reuse these variables, we should make this change, but I think it is easier to understand currently since it's just the one file.

Reply:
> > I feel like this should be in an .env file
> 
> I think we as the scope grows and we find ourselves needing to reuse these variables, we should make this change, but I think it is easier to understand currently since it's just the one file.

Can we talk about this during a standup?